tests/test_restore_names.py

```diff
@@ -1,3 +1,65 @@
 from app.restore_names import restore_names
+import pytest
 
-# write your tests here
+
+@pytest.mark.parametrize(
+    "lost_user_data,restored_user_data",
+    [
+        pytest.param(
+            [
```

Comment:
Could you create separate tests for both of the cases? 
It'd be easier to debug function in case of bugs